GameData/SPEngine/Parts/NK9_proc.cfg

```diff
@@ -7,13 +7,13 @@
 	MODULE
 	{
 		name = ModuleSPEngine
-		familyLetter = n
+		familyLetter = N
```

Comment:
No.  The n-class and N-class are two different families; the NK-9 is an n-class.  The N-class is the NK-33, a much larger and more powerful engine.